test/routes/tsoa/authorize.spec.ts

```diff
@@ -515,13 +519,16 @@ describe("authorize", () => {
       const redirectUrl = new URL(locationHeader);
       const searchParams = new URLSearchParams(redirectUrl.hash.slice(1));
 
-      const idToken = parseJwt(searchParams.get("id_token") as string);
+      const { sid, sub, ...idToken } = parseJwt(
+        searchParams.get("id_token") as string,
+      );
+
+      expect(sub).toMatch(/^email|/);
+      expect(sub).toBeTypeOf("string");
```

Comment:
ahhh, I get it now

```suggestion
      expect(sid).toBeTypeOf("string");
```

It should be `sid`